code/datums/abilities/critter/ocean.dm

```diff
@@ -19,10 +19,6 @@
 		var/obj/overlay/tile_effect/cracks/C = new(T)
 		holder.owner.set_loc(C)
 
-		if (holder.owner.ai)
-			holder.owner.ai.disable()
```

Comment:
It will, but it will also mean that they won't just bury themselves and wait to be triggered for the whole round. Evaluation of mobAI is very light now, so it shouldn't be an issue.

Reply:
Why not just re-enable the AI when they're triggered though?